src/ucp/core/ucp_rkey.c

```diff
@@ -458,6 +459,17 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_ep_rkey_unpack_internal,
 
         ucs_assert(rkey_index < md_count);
         tl_rkey       = &rkey->tl_rkey[rkey_index];
+
+        if (UCS_BIT(remote_md_index) & skip_md_map) {
```

Comment:
we need to distinguish MDs that we don't want to unpack at all and those which are "stubbed"

Reply:
per offline discussion, decided to change the usage of "unpack_md_map" to be "~skip_md_map"; use ep->reachable_md_map as unpack_md_map; and peer mem get will unpack uct rkey instead of ucp rkey

Reply:
@yosefe, per our offline discussion using UCT rkey would still need to have some common code accepting both types of map - skip and stub. Or we will need to create a separate func working with rkey parsing, which is error prone.